src/usdt/mod.rs

```diff
@@ -0,0 +1,220 @@
+use std::{cell::RefCell, os::raw::c_char};
```

Comment:
Please change this to two imports. You can combine std::os::raw::c_char with the one for c_int